src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        write(toFileName, report(buyAndSupply(read(fromFileName))));
+    }
+
+    private String[] read(String fileName) {
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            String value = reader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(" ");
```

Comment:
What if we add a `comment` column and some cells contain `We need to buy more apples`?